tests/acceptance/dependency-test.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from 'ember-twiddle/tests/helpers/start-app';
+
+let application;
+
+module('Acceptance | external dependency', {
+  beforeEach: function() {
+    application = startApp();
+  },
+
+  afterEach: function() {
+    Ember.run(application, 'destroy');
```

Comment:
@Gaurav0 - To clarify, @stefanpenner is referring to avoiding usage of the `var application` variable here.  Save the created instance on `this` instead (helps the test framework ensure proper cleanup is done).